extensions/wikia/UserLogin/UserLoginHelper.class.php

```diff
@@ -401,10 +396,11 @@ public function incrMemc( $memKey, $period=86400 ) {
 
 	/**
 	 * clear password-throttle cache
+	 *
 	 * @param string $username
 	 */
 	public function clearPasswordThrottle( $username ) {
-		$key = wfMemcKey( 'password-throttle', wfGetIP(), md5($username) );
+		$key = wfMemcKey( 'password-throttle', $this->wg->Request->getIP(), md5( $username ) );
```

Comment:
I was thinking the whole line because it is repeated.
```php
protected function getPasswordThrottleCacheKey( $username ) {
	return wfMemcKey( 'password-throttle', $this->wg->Request->getIP(), md5( $username ) );
}
```